api/models/ssl.go

```diff
@@ -31,7 +31,7 @@ type SSL struct {
 
 type SSLs []SSL
 
-func CreateSSL(app, process string, port int, body, key string, chain string, secure bool) (*SSL, error) {
+func CreateSSL(app, process string, port int, arn string, body, key string, chain string, secure bool) (*SSL, error) {
```

Comment:
Wasn't clear to us immediately which args should have types attached to them (and why `body` doesn't have one).